app/webpack/packs/unlink.js

```diff
@@ -0,0 +1,27 @@
+function Unlinking () {
+  // rely on `name`` because id changes in error state
+  //
+  const unlinkReasonCode = document.getElementsByName('unlink_attempt[reason_code]')[0]
+  const unlinkOtherReasonText = document.getElementsByName('unlink_attempt[other_reason_text]')[0]
+  const otherReasonCode = '7'
+
```

Comment:
@jsugarman you may wish to add `if (!unlinkReasonCode) { return false }` before line 8.

The return statement executes if the element is undefined, it therefore stops the rest of the code from running, this helps performance and reduces errors